### PR TITLE
Update k6 installation instructions and GitHub repo URL

### PR DESCRIPTION
**Pull request by imiric**
Let me know if you prefer the repo URL changes in a separate PR, otherwise you can review by commit.

I also changed the Spanish translation of the installation page, so proofreading by @ppcano is welcome. :)

### CODE REVIEW

File: src/data/markdown/translated-guides/en/01 Getting started/02 Installation.md
Comment (na--) [inline src/data/markdown/translated-guides/en/01 Getting started/02 Installation.md:73]: Since we haven't taken over this package yet, we should add a disclaimer here that it's not an official package.

**Review by na-- — APPROVED** [review]
LGTM besides the minor nitpick about Chocolatey, and it's awesome you can just directly update the Spanish translations :tada:

Please don't merge this yet, I want to manually test the new repos for a bit before we merge this and https://github.com/k6io/k6/pull/1978

**Review by MStoykov — APPROVED** [review]

File: src/data/markdown/translated-guides/en/01 Getting started/02 Installation.md
Comment (imiric) [reply src/data/markdown/translated-guides/en/01 Getting started/02 Installation.md]: Sure, see cbf354f.

**Pull request merged**
